packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -562,17 +564,40 @@ function PayeeCell({
         }
       }}
       formatter={() => getPayeePretty(transaction, payee, transferAcct)}
-      unexposedContent={props => (
-        <>
-          <PayeeIcons
-            transaction={transaction}
-            transferAccount={transferAcct}
-            onNavigateToTransferAccount={onNavigateToTransferAccount}
-            onNavigateToSchedule={onNavigateToSchedule}
-          />
-          <UnexposedCellContent {...props} />
-        </>
-      )}
+      unexposedContent={props => {
+        const payeeInfo = (
+          <div style={{ display: 'flex', alignItems: 'center', width: '100%' }}>
```

Comment:
Is this the VRT issue?  You set the width to 100% here so the children will fill it?

Reply:
Most likely, I have no idea why I did that... must have been a reason. Will check when I can get back to a dev box.